shared_model/interfaces/permission_to_string.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PERMISSION_TO_STRING_HPP
+#define PERMISSION_TO_STRING_HPP
+
+#include <string>
+#include <vector>
+
+#include "interfaces/permissions.hpp"
+
+namespace shared_model {
+  namespace interface {
+    class PermissionToString {
```

Comment:
I don't like name `PermissionToString` for a class :( I think, it should some noun, for example, `PermissionStringifier` or `PermissionConverter`, though the second variant is less precise

Reply:
`PermissionStringifier` sounds even uglier :( That class is just a method collection w/o any actual data. So imo the name is fine